server/src/internalClusterTest/java/org/elasticsearch/search/aggregations/pipeline/AvgBucketIT.java

```diff
@@ -86,21 +89,29 @@ public void setupSuiteScopeCluster() throws Exception {
         }
         indexRandom(true, builders);
         ensureSearchable();
+        histoName = randomName();
+        termsName = randomName();
+    }
+
+    private static String randomName() {
+        return randomBoolean()
+            ? randomAlphaOfLengthBetween(3, 12)
+            : randomAlphaOfLengthBetween(3, 6) + "." + randomAlphaOfLengthBetween(3, 6);
```

Comment:
We should test non-ascii characters too, probably in a follow up PR.  Or just open a ticket for it, could be a good first issue for someone.